lib/services/sshd.pm

```diff
@@ -0,0 +1,31 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+# Summary: Package for ssh service tests
+#
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package services::sshd;
+use base 'opensusebasetest';
+use testapi;
+use utils;
+use strict;
+use warnings;
+
+
+sub check_sshd_ports_opened {
```

Comment:
both protocols use the same port, could we find another name? for example `check_sshd_port`?

Reply:
makes sense